config/crd/bases/workload.nephio.org_nfdeployments.yaml

```diff
@@ -314,8 +251,28 @@ spec:
                   - name
                   type: object
                 type: array
+              parametersRefs:
+                description: configRef defines addiitonal configuration references
```

Comment:
"parametersRef defines additional"

Reply:
can you approve this PR https://github.com/nephio-project/api/pull/53
it addresses your comment